test/unit/host_test.rb

```diff
@@ -148,7 +148,7 @@ class HostTest < ActiveSupport::TestCase
   end
 
   test "can fetch vm compute attributes" do
-    host = FactoryGirl.create(:host, :compute_resource => compute_resources(:one))
+    host = FactoryGirl.create(:host, :compute_resource => compute_resources(:ec2))
```

Comment:
`:one` is libvirt which handles the compute parameters a bit differently so I switched it to `:ec2`. I'll add unit tests for the other compute resources too.